### ISSUE TITLE
scheduler with parameter is not supported

### ISSUE THREAD

**Issue by 2001092236** [r-isachenko/dgm_utils#3]
https://github.com/r-isachenko/dgm_utils/blob/9c3ef18c821268297f31d45e36c3b0f2bb538a80/dgm_utils/training.py#L86C1-L88C2

add the following code:

if scheduler is not None:
    try:
        scheduler.step()
    except:
        scheduler.step(train_loss)

**Comment by gregkseno** [comment]
Тебе это нужно для этого ReduceLROnPlateau планировщика?
